portfolio/cloudbuild.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
+ - name: maven:3-jdk-8
+   entrypoint: mvn
+   args: ["package" "appengine:deploy"]
```

Comment:
Is this a duplicate of the file included above?

Reply:
I moved it from the root to the portfolio directory

Reply:
It looks like this file and `portfolio/cloudbuild.yaml` are still included in this PR. is that intentional?

Reply:
No, I've gone ahead and removed it